app/views/comments/_form.html.erb

```diff
@@ -11,22 +11,35 @@
     </div>
   <% end %>
 
+  <!-- new code below -->
+
+  <%= form.hidden_field :photo_id %>
+
+
```

Comment:
Make sure to take out the whitespaces when making changes. But well done implementing the hidden forms. Good consideration for the user experience.